CMakeLists.txt

```diff
@@ -192,9 +195,11 @@ else()
     add_definitions(-DDMLC_USE_CXX11=1)
     add_definitions(-DDMLC_USE_CXX14=1)
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14")
+    set(CMAKE_CUDA_STANDARD 14)
```

Comment:
It's already set in line 93 https://github.com/apache/incubator-mxnet/blob/ee31ae54bf7099c6ceda4a4bd1d7ccd8c431dea9/CMakeLists.txt#L93